pages/repo/edit.js

```diff
@@ -415,14 +435,16 @@ export class EditorPage extends Component {
 
     const { schema } = this.state
     if (!schema) {
-      const commit = repo && repo.commit
+      const commit =
+        (repo && repo.commit) || (templateRepo && templateRepo.latestCommit)
+
+      const schema =
+        (commit && commit.document.meta.template) || router.query.schema
```

Comment:
Let's add a fallback for old (open) urls:
```suggestion
      const schema =
        (commit && commit.document.meta.template) || router.query.schema || router.query.template
```